### DISCUSSION TITLE
Error: YoutubeExplode.Exceptions.VideoUnavailableException

### DISCUSSION THREAD

**Discussion by lukaszfiszer96** [Tyrrrz/YoutubeExplode#809] [Q&A]
Hi, 
dll work fine until 08.09. Then I got YoutubeExplode.Exceptions.VideoUnavailableException: Video 'G1Xjzd97d_E' is not available.

Problem exist only on azure deployed app. Locally works fine.

**Comment by yousufanwar** [comment]
i'm facing same issue, deployed app is not downloading, locally works fine
